docs/user/gcp/install_upi.md

```diff
@@ -24,6 +24,12 @@ Use a [staged install](../overview.md#multiple-invocations) to enable desired cu
 ### Create an install config
 Create an install configuration as for [the usual approach](install.md#create-configuration).
 
+If you are installing into a [Shared VPC (XPN)][sharedvpc],
+the installer will not be able to find the base domain zone in the host project.
+Create the `install-config.yaml` manually using the documentation references/examples.
+It must have `publish: Internal` to avoid the base domain verification when executing `create manifests` later in this workflow.
```

Comment:
```suggestion
Create the `install-config.yaml` manually using the documentation references/examples.
It must have `publish: Internal` because the installer will not be able to find the public DNS zone for the the base domain in the host project, which is required for the External clusters.
```

Reply:
ptal 67658b5fca0aeaa92b80e428086fdd30037dfc9d